pkg/noun/manage.c

```diff
@@ -617,11 +617,16 @@ _find_home(void)
     nor_w = (low_p + ((1 << u3a_page) - 1)) >> u3a_page;
     sou_w = u3P.pag_w - (hig_p >> u3a_page);
 
-    if ( (nor_w != u3P.nor_u.pgs_w) || (sou_w != u3P.sou_u.pgs_w) ) {
+    if ( (nor_w > u3P.nor_u.pgs_w) || (sou_w > u3P.sou_u.pgs_w) ) {
       fprintf(stderr, "loom: corrupt size north (%u, %u) south (%u, %u)\r\n",
                       nor_w, u3P.nor_u.pgs_w, sou_w, u3P.sou_u.pgs_w);
       u3_assert(!"loom: corrupt size");
     }
+
+    if ( (nor_w < u3P.nor_u.pgs_w) || (sou_w < u3P.sou_u.pgs_w) ) {
```

Comment:
means: (north|south) road are smaller than the (north|south) snapshot. Why is that not an error? When should the size of a road's snapshot be allowed to exceed the road itself when applying it?

Reply:
It's strange, and shouldn't happen, so we print a warning. But it's possible that the snapshots are otherwise correct, and we don't want to prevent their restoration in that case.

(As i'm writing this, I'm realizing that's only true for north.bin -- south.bin is reversed on disk and always 0-1 pages, so any mismatch will be fatal. I'll revise)